docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
 .. _release_and_maintenance:
 
-Release and maintenance
-=======================
+************************
+Releases and maintenance
+************************
 
-This section describes the Ansible and ``ansible-core`` releases.
-Ansible is the package that most users install. ``ansible-core`` is
-primarily for developers.
+This section describes release cycles, rules, and maintenance schedules for both Ansible community projects: the Ansible community package and ``ansible-core``. The two projects have different versioning systems, maintenance structures, contents, and workflows.
+
+====================================================  =================================================
+Ansible community package                             ansible-core
+====================================================  =================================================
+each major release is a new major version (4.0.0)     continues "classic Ansible" versioning (2.11)
+follows semantic versioning rules                     does not use semantic versioning
+maintains only one version at a time                  maintains latest version plus two older versions
+includes language, runtime, and selected Collections  includes only the language and runtime
+developed and maintained in Collection repositories   developed and maintained in ansible/ansible repo
+====================================================  =================================================
+
+Most community users install the Ansible community package. The ``ansible-core`` option is primarily for developers.
```

Comment:
Is this really the message we want to send out? I see the `ansible-core` package plus needed collections as the way forward. The "we include everything and the kitchen sink" approach is in my books a stopgap measure that gives users time to migrate their stuff.

Reply:
@tadeboro what do you think of https://github.com/ansible/ansible/pull/73781#discussion_r587332303 ?